org.hl7.fhir.r5/src/test/java/org/hl7/fhir/r5/test/CanonicalResourceManagerTests.java

```diff
@@ -927,4 +929,877 @@ public void testSupplements() {
     sl = mrm.getSupplements("http://url/CodeSystem/s234");
     Assertions.assertEquals(0, sl.size());
   }
-}
\ No newline at end of file
+
+
+  @Test
+  public void testVersionContentionTemplate() throws ParseException {

```

Comment:
Starting from this test here, there is a LOT of boilerplate for what seems to amount to five args:

codeSystem1, package1, codeSystem2, package2, expectedCodeSystemId

This seems to involve the majority of the generated tests, which would be much cleaner if parameterized.